### ISSUE TITLE
Support incremental unzipping (skip already-extracted zip files)

### ISSUE THREAD

**Issue by Rob--W** [cr/webextaware#27]
`webextaware unzip all -o /path/to/dir/` unzips all extensions to the given directory.

When `unzip` is called again, it attempts to unzip again even if the files already exists.

It would be nice if the tool can skip extracting zipfiles when the output directory (`/path/to/dir/id/hash/`) is non-empty.

E.g. by checking whether `self.unzip_folder` is non-empty, here:
https://github.com/cr/webextaware/blob/b504cee6c76131010af9ddcdf77d0f4fe03d093b/webextaware/webext.py#L64

Maybe via a command line flag at that is forwarded at:
https://github.com/cr/webextaware/blob/b504cee6c76131010af9ddcdf77d0f4fe03d093b/webextaware/modes/unzip.py#L51

**Comment by cr** [comment]
I considered this on implementation. For it to work reliably, you'd need the zipfile module to take care of overwriting on a file-by-file basis on extraction – which it does not – or implement your own per-file extraction logic. An "is empty" check would result in issues with half-extracted archives etc. Then there's the question of how to handle locally changed files etc.

In short, it either seemed half-baked and brittle, or it would require significantly more complex code for a rarely-used function that takes just a few minutes to complete anyway, so I didn't bother spending the time.

However, I do see the benefits for when the command is used regularly for updating an extracted extension pool, and I'd add the "-n --nooverwrite" flag. However, I wouldn't want to pretend to be smart with some "is directory empty" logic, but dumb it down even further to "does directory exist" instead.

**Comment by cr** [comment]
I'll push the new release to pypi momentarily.
